src/components/App.js

```diff
@@ -1,13 +1,12 @@
 import React from 'react'
 
 import { Root, Container } from './Common'
-import { Context } from '../context'
+// import { Context } from '../context'
 import Content from './Content'
 import Header from './Header'
 
 export default function Page() {
-  const state = React.useContext(Context)
-  console.log({ state })
+  // const state = React.useContext(Context)
```

Comment:
is this left in as a comment to show others that it's ready to use?

Reply:
yes!